_posts/2017-10-10-C4Q-QandA.markdown

```diff
@@ -1,15 +1,14 @@
 ---
 layout: epic
-title: "C4Q Q&A"
+title: "C4Q&A"
 date: 2017-10-10
 author: [orta, craig, kana, db, xtina]
 categories: [culture, teaching, event, panel]
-series: Swift at Artsy
 ---
 
-C4Q is a non-for-profit hacker school based in NYC. We've had members of Artsy team help out by being TAs, run 
+[C4Q is](https://www.c4q.nyc) a non-for-profit hacker school based in NYC. We've had members of Artsy team help out by being TAs, run 
 committees and steer the curriculum as Engineers in the industry. C4Q recently reached out to arrange a meetup
-between two of their classes learning Android and Web and Artsy engineers. 
+between for half of students who are learning Android and Web with Artsy engineers. 
```

Comment:
I'm having trouble parsing this first part, maybe something like:

> C4Q is a non-for-profit hacker school based in NYC. We've had members of the Artsy team help out by being TAs, running committees and steering the curriculum as Engineers in the industry for many years. C4Q recently reached out to arrange a meetup between Artsy engineers and students who are learning Android and Web development.

I think it's ok to leave out the part about how many of their students were involved.